post-install.yml

```diff
@@ -102,6 +102,16 @@
         git push {{ dashboard_git_remote.stdout }} master
       args: 
         chdir: /tmp/tsuru-dashboard
+      register: dashboard_push_result
+
+    - name: Tsuru set environment variables for dashboard
+      shell: >
+        tsuru env-set -a dashboard STATSD_PORT=8125;
+        tsuru env-set -a dashboard STATSD_HOST={{ deploy_env }}-metrics.{{ domain_name }};
+        tsuru env-set -a dashboard GRAPHITE_HOST={{ deploy_env }}-metrics.{{ domain_name }};
+        tsuru env-set -a dashboard GRAPHITE_USER={{ graphite_auth_user }}
+        tsuru env-set -a dashboard GRAPHITE_PASS={{ graphite_auth_pass }}
+      when: "not 'Everything up-to-date' in dashboard_push_result.stderr"
```

Comment:
If we set these variables before we first push the dashboard, then it won't need to restart the dashboard after it's been deployed for it to read the newly defined variables - can we move them up to happen before the git push ?